### ISSUE TITLE
cobbler xenserver6.2.0 after pxe error after #589

### ISSUE THREAD

**Issue by Johnsmith1111** [cobbler/cobbler#1371]
cobbler version is 2.6.3
I import Xenserver-6.2.0 in cobbler in my vmvare workstation and the import is ok(refer #589), 
but when I enter the pxe in another virtual machine and enter the XenServer-6.2.0-x86_64 option,
it turns vmvare-error as :
vcpu-0:This virtual machine has tried to execute an invalid part of memory. It is possible that the virtual disk or CD media you are using is not bootable or that a program or driver running in the virtual machine has malfunctioned.
![cobblererror](https://cloud.githubusercontent.com/assets/6069740/5406583/62d1a596-81f0-11e4-903c-a910ae74232f.jpg)
and in the virtual machine it after the message it goes down,I dont known why,
but I directly import the iso as CD in my virtual machine and it works. 
and I dont know where the TFTP BOOT files is ,So I cant add the line metion in #589  after the install.img line:
`$local_img_path/vmlinuz=/var/www/cobbler/ks_mirror/XenServer-6.2.0/boot/vmlinuz`

**Comment by bms** [comment]
No, these look like separate issues to me. I'm still fighting the signature parser.
